common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -559,6 +559,41 @@ def test_navigation_buttons(self):
             sequence_ui_events
         )
 
+    def test_accordion_events(self):
+        # navigate to second subsection via accordion
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1,2')
+
+        # navigate to first section of next chapter via accordion
```

Comment:
Are these comments necessary or superfluous?  I recommend removing them since it's evident from the method call name of what it's doing.  For example, in the future, once we remove the accordion, the underlying implementation of the method may change, but this comment would be outdated.